### DISCUSSION TITLE
Would love to learn Java/modding

### DISCUSSION THREAD

**Discussion by dwi11y** [FabricMC/fabric-api#4307] [Mod Dev Support]
I'm trying to put together a mod pack and would like to know if I could fix the following issue on my end

 !![image](https://github.com/user-attachments/assets/cea2638e-9e3b-4103-a45b-aabecce84254)

My game is running fine,I just would like to learn more. Thank you

Edit: Im confident the mod causing the issue is "YACL"

**Comment by Cai-kunkun** [comment]
That is not a big problem.
Actually, it won't make the game crash.
Have a good time!

**Reply by dwi11y** [reply]
Tysm

**Reply by Cai-kunkun** [reply]
ur welcome

**Reply by Cai-kunkun** [reply]
You can close this discussion right now.
Thank you!
